### PR TITLE
Fixed SwiftLint warnings #554 Issue

### PR DESCRIPTION
**Pull request by alexmomotiuk**
Fixed SwiftLint warnings #554

### CODE REVIEW

**Comment by SD10** [comment]
Thanks for submitting this @ProgiiX. I'm guessing the build is failing because of the setup of CircleCI on the development branch. We'll have to get `master` merged into `developement` before we can merge this

**Comment by SD10** [comment]
@ProgiiX You're going to have to rebase this pull request with the latest changes on `development`. Sorry for the inconvenience.

**Comment by SD10** [comment]
@ProgiiX You don't need to create new pull requests. You can just push the changes to the same branch and these will automatically update 😛

**Comment by alexmomotiuk** [comment]
@SD10 Thanks. I pushed changes from development branch, but build is failing.

**Comment by SD10** [comment]
Thanks @ProgiiX. I'm not sure why this is failing. I triggered a rebuild. Let's see what happens this time.

File: Sources/Views/MessageInputBar.swift [hunk 1]
```diff
@@ -573,14 +575,18 @@ open class MessageInputBar: UIView {
     ///   - animated: If the layout should be animated
     open func setStackViewItems(_ items: [InputBarButtonItem], forStack position: InputStackView.Position, animated: Bool) {
         
+        func setupStackViewItem(_ item: InputBarButtonItem) {
```
Comment (SD10) [inline Sources/Views/MessageInputBar.swift]: Do you mind if we revert this change? I'd rather not add it if it only trims a single line

Comment (alexmomotiuk) [reply Sources/Views/MessageInputBar.swift]: No, I don't mind.

**Review by SD10 — APPROVED** [review]
Thanks @ProgiiX 💯 👍

**Comment by SD10** [comment]
Thank you for contributing to MessageKit! I've invited you to join the MessageKit GitHub organization - no pressure to accept! If you'd like more information on what that means, check out our [contributing guidelines](https://github.com/MessageKit/MessageKit/blob/master/CONTRIBUTING.md) and join the [MessageKit Slack channel](https://join.slack.com/t/messagekit/shared_invite/enQtMjI4NzIzNzMyMzU0LTEyNmYzNzBlNjNiYjAwYjY1ZjhkMmVjNDE4OGNlMDY1OTRmNzVmMzI4MTg3OWQzZTM2NGU4NTU3ZDE5NzkwNzQ). Feel free to reach out if you have any questions! :smiley:

**Comment by alexmomotiuk** [comment]
@SD10 Thanks :)
